### ISSUE TITLE
Unable to run Fedora 28,27 on Windows 10 v1803

### ISSUE THREAD

**Issue by gaurav-nelson** [DDoSolitary/LxRunOffline#46]
With the latest version on Windows (1803) I am able to install Fedora 27, 28 but unable to run `dnf update` and other commands. However, my other fedora 27 instance, which I created before the update is working fine.

```
[root@PC]# dnf update
Segmentation fault (core dumped)
[root@PC]# dnf update
error: db5 error(12) from dbenv->open: Cannot allocate memory
error: db5 error(22) from dbenv->close: Invalid argument
error: cannot open Packages index using db5 - Cannot allocate memory (12)
error: cannot open Packages database in /var/lib/rpm
Error: Error: rpmdb open failed
[root@PC]# cat /etc/*-release
Fedora release 28 (Twenty Eight)
NAME=Fedora
VERSION="28 (Twenty Eight)"
ID=fedora
VERSION_ID=28
PLATFORM_ID="platform:f28"
PRETTY_NAME="Fedora 28 (Twenty Eight)"
ANSI_COLOR="0;34"
CPE_NAME="cpe:/o:fedoraproject:fedora:28"
HOME_URL="https://fedoraproject.org/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=28
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=28
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
Fedora release 28 (Twenty Eight)
Fedora release 28 (Twenty Eight)
```
Any help appreciated.

**Comment by DDoSolitary** [comment]
Please post the output of the failing commands, thanks.

**Comment by gaurav-nelson** [comment]
Thanks @DDoSolitary for looking into this, I have updated the issue with logs. 

- I followed the steps in https://medium.com/@gauravnelson/how-to-run-fedora-with-windows-subsystem-for-linux-wsl-a148902e5087 to get it running successfully before the update.  
- I also tried to run it using https://github.com/Microsoft/WSL-DistroLauncher but I get the same errors. 

I am guessing it has to do something with the kernel version maybe? I am not sure yet.
